numpy/core/src/common/simd/vsx/arithmetic.h

```diff
@@ -118,6 +118,28 @@
 
 // Horizontal add: Calculates the sum of all vector elements.
 
+NPY_FINLINE npy_uint16 npyv_sumup_u8(npyv_u8 a)
+{
+    const npyv_u32 zero = npyv_zero_u32();
+    npyv_u32 four = vec_sum4s(a, zero);
+    npyv_u32 one  = vec_sums((npyv_s32)four, (npyv_s32)zero);
```

Comment:
```suggestion
    npyv_s32 one  = vec_sums((npyv_s32)four, (npyv_s32)zero);
```

Reply:
I think this is what the CI failure comes from.